geometrical_calculator_kuba.py

```diff
@@ -0,0 +1,393 @@
+import os
+import csv
+import math
+
+geometrical_calculator_history = []
+
+class Circle:    
+    
+    def __init__(self , r):
+        self.r = r
+                
+    def calculate_area(self):
+        return math.pi * self.r ** 2
+    
+    def calculate_circumference(self):
+        return 2 * math.pi * self.r
+    
+class Triangle:
```

Comment:
Ok, na tym przykładzie. 

`class Triangle:
    def __init__(self, side_a, side_b, side_c, height=None, base=None):
        self.side_a = side_a
        self.side_b = side_b
        self.side_c = side_c
        self.height = height
        self.base = base

    def calculate_area(self):
        if self.base is None or self.height is None:
            raise ValueError("Tu zwracasz, że dane są wymagane do obliczeń.")
        return 0.5 * self.base * self.height

    def calculate_circumference(self):
        return self.side_a + self.side_b + self.side_c
`